public/btrfs-progs/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+NOCONFIGURE=1 ./autogen.sh
+
+# Requires losetup which is part of util-linux, but even with it installed it fails
+rm -rf -- tests/mkfs-tests/017-small-backing-size-thin-provision-device
```

Comment:
I don’t think — is needed here.

Reply:
So if I remove it the above comment is obsolete, isn't it?

Reply:
Just remove the “—“. It’s not needed.

Reply:
Ahh I see. Will do all changes asap. I'm currently out walking with my dog.